src/licensedcode/data/licenses/smail_gpl.yml

```diff
@@ -0,0 +1,8 @@
+key: smail_gpl
+short_name: SMAIL GPL
+name: SMAIL GENERAL PUBLIC LICENSE
+category: Copyleft
+owner: Landon Curt Noll & Ronald S. Karr
+homepage_url: https://gitweb.gentoo.org/repo/gentoo.git/plain/licenses/
```

Comment:
it is ok here not to have a homepage so I would remove this rather than to make it up